backend/src/routes/user.ts

```diff
@@ -364,4 +366,83 @@ router.get(
   }
 );
 
+/**
+ * Route to setup mobile app for any logged in user (mentor or mentee)
+ * 
+ * This route returns the following
+ * If user is a mentor, 
+ *  menteeIds, status, upcomingSessionId
+ * 
+ * If user is mentee,
+ *  mentorId, status, upcomingSessionId
+ */
+router.get(
```

Comment:
@adhi0331 created a route to get all the data that's useful for navigating between no session vs 1 session so that the `SessionDetailsPage` would need minimal changes for final version. 

With your home page implementation, feel free to get rid of this route or edit on top of it